src/02-data-types.js

```diff
@@ -49,7 +55,9 @@ function convertToString(val) {}
  *  convertToShoutingText("Hello There");
  *  //> "HELLO THERE"
  */
-function convertToShoutingText(text) {}
+function convertToShoutingText(text) {
+  return text.toUpperCase()
```

Comment:
Something we are missing on much of our code are ` ; ` marks at the end of our statements.  Lets' add them at the end of our statements and at the end of our ` return `s